module-utils/utility/tests/unittest_utils.cpp

```diff
@@ -163,18 +163,129 @@ TEST_CASE("Converts enum to string")
 
 TEST_CASE("Get value from string")
 {
-    SECTION("UInt32_t")
+    SECTION("uint8_t")
+    {
+        std::string testString = "10";
+        const auto testValue   = utils::getNumericValue<uint8_t>(testString);
+        REQUIRE(testValue == 10);
+    }
+
+    SECTION("std::uint8_t")
+    {
+        std::string testString = "10";
+        const auto testValue   = utils::getNumericValue<std::uint8_t>(testString);
```

Comment:
What's the point of checking for both `uint8_t` and `std::uint8_t`?. They are the same ;)

Reply:
The same applies to the other cases.

Reply:
I just wanted to be absolutely sure it'll work as designed ;) Removed the redundant tests.